src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -181,7 +183,11 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
   })
 
   const getWithdrawGasEstimateCryptoBaseUnit = useCallback(
-    async (withdraw: WithdrawValues) => {
+    async (
+      withdraw: WithdrawValues,
+      quote: ThorchainSaversWithdrawQuoteResponseSuccess,
+      dustAmountCryptoBaseUnit: string,
+    ): Promise<Result<string, string> | null> => {
```

Comment:
`Result` monad saving the day again, as this can be null (i.e `None` i.e nothing to see yet), `Some` (i.e `Ok` i.e happy result) or `Err` (i.e sad result)